single-pool/js/packages/client/src/quarantine.ts

```diff
@@ -0,0 +1,264 @@
+import {
+  getBase58EncodedAddressCodec,
+  Base58EncodedAddress,
+  AccountRole,
+  getProgramDerivedAddress,
+} from '@solana/web3.js';
+
+// HERE BE DRAGONS
+// this is all the stuff that shouldnt be in our library once we can import from elsewhere
+
+function address<TAddress extends string>(string: TAddress): Base58EncodedAddress<TAddress> {
+  return string as Base58EncodedAddress<TAddress>;
+}
+
+export const SYSTEM_PROGRAM_ID = address('11111111111111111111111111111111');
+export const STAKE_PROGRAM_ID = address('Stake11111111111111111111111111111111111111');
+export const SYSVAR_RENT_ID = address('SysvarRent111111111111111111111111111111111');
+export const SYSVAR_CLOCK_ID = address('SysvarC1ock11111111111111111111111111111111');
+export const SYSVAR_STAKE_HISTORY_ID = address('SysvarStakeHistory1111111111111111111111111');
+export const STAKE_CONFIG_ID = address('StakeConfig11111111111111111111111111111111');
+export const STAKE_ACCOUNT_SIZE = 200n;
+
+export const TOKEN_PROGRAM_ID = address('TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA');
+export const ATOKEN_PROGRAM_ID = address('ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL');
+export const MINT_SIZE = 82n;
+
+export function u32(n: number): Uint8Array {
+  const bns = Uint32Array.from([n]);
+  return new Uint8Array(bns.buffer);
+}
+
+export function u64(n: bigint): Uint8Array {
+  const bns = BigUint64Array.from([n]);
+  return new Uint8Array(bns.buffer);
+}
+
+export class SystemInstruction {
```

Comment:
cc/ @lorisleiva re: @2501babe wanting some off-the-shelf, autogenerated clients for use here to talk to the system program, token program, and stake program.